src/action.c

```diff
@@ -679,15 +679,15 @@ show_menu(struct server *server, struct view *view,
 
 		if (!strcasecmp(pos_x, "center")) {
 			x = (output->usable_area.width / 2) - (max_width / 2);
-		} else if (strcasestr(pos_x, "%")) {
+		} else if (strstr(pos_x, "%")) {
```

Comment:
I think `strchr()` would be even better.

Reply:
All I'm doing is searching for a "%", I'm not doing anything with the position, as atoi will stop at the first non digit character. Either way should work. Easy fix.